LDAP/env.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+#
+# Set this to the Sysdig Monitor API Token value shown in the Sysdig UI under
+# Settings->User Profile when logged in as the "Super" Admin User. For
+# information on locating this user, see the following article:
+#
+# https://support.sysdig.com/hc/en-us/articles/115004951443-Locating-the-Super-Admin-User
+#
+export API_TOKEN="aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+
+#
+# Set this to the URL through which you access your Sysdig application UI.
+#
+export URL="https://10.0.0.1"
+
+#
+# Leave this set to "-k" to allow curl to connect to your Sysdig API even if a self-
+# signed certificate is in use (the default in a Sysdig software platform install).
+#
+export CURL_INS="-k"
+#export CURL_SEC=""
+
+#
+# Change this to "-v" if you want curl to print verbose output, such as for debugging.
+# The "-s" keeps curl in silent mode to make outputs brief.
+#
+export CURL_VERBOSITY="-s"
+#export CURL_VERBOSITY="-v"
+
```

Comment:
It might be nice to set a single CURL_OPTS that combines all the above settings to be used later

Reply:
@mbreitung: Good call. I've consolidated everything to a single `CURL_OPTS`, and also added your useful suggestion of `-w \n`. I've committed & re-tested with the examples from the customer-facing doc.